client/signup/config/flows-pure.js

```diff
@@ -145,7 +145,7 @@ export function generateFlows( {
 			destination: getSignupDestination,
```

Comment:
Wondering if we can completely remove `plans-pm` and replace it with `plans` above? So that the `steps` array is `[ 'user', 'domains', 'plans' ],`?

Reply:
@aneeshd16 Although we are trying to bring the two flows closer together we also need some wiggle room to deviate and do changes where required. This is why plans-pm remains a different step.

Reply:
Yeah, like @jdc91 said :) The main reason that it's there is because we need to pass `showBiennialToggle: true` prop. The current way configured this prop right in the step configuration, so it has to be a separate step.

Reply:
Thanks for the clarification!